apps/docs-e2e/src/docs.spec.ts

```diff
@@ -150,23 +154,287 @@ test.describe('should contain docs page elements', {
   })
 })
 
-// TODO: complete test suite for mobile
+// test suite for mobile layout
```

Comment:
Try this:

![image](https://github.com/user-attachments/assets/9f266dae-92fd-4a90-99a5-18300b7aae3d)

You can find another example of me using dividers in [fumadocs.global.css](https://github.com/cuhacking/2025/blob/main/apps/docs/src/app/docs/fumadocs.global.css)